.github/workflows/amzn2.yml

```diff
@@ -0,0 +1,90 @@
+name: Update BTFHub Archive for Amazon Linux 2
+on:
+  schedule:
+    - cron: "0 1 * * *"
+  workflow_dispatch: {}
+jobs:
+  amazon-update:
+    name: update amazon linux 2 btfhub-archive
+    runs-on: ubuntu-latest
+    container:
+      image: amazonlinux:2
+    steps:
+      - name: Install needed amazon packages
+        run: |
+          yum install -y yum-utils wget tar gzip xz clang make cmake git libdwarf-devel elfutils-libelf-devel elfutils-devel
```

Comment:
Version of clang does not matter for our usecase?

Reply:
It does, `btfhub` doesn't compile with the older version of clang that is available via a `yum` package, and conversely I was having trouble getting `elfutils` and `tar` building with the newer clang (probably because I did a manual install).